lib/ansible/modules/packaging/os/rhn_channel.py

```diff
@@ -146,8 +130,8 @@ def main():
     user = module.params['user']
     password = module.params['password']
 
-    #initialize connection
-    client = xmlrpclib.Server(saturl, verbose=0)
+    # initialize connection
+    client = xmlrpc_client(saturl)
```

Comment:
Same as in https://github.com/ansible/ansible/pull/26878, `xmlrpc_client.ServerProxy(saturl)`?